### ISSUE TITLE
Routify Ignore File

### ISSUE THREAD

**Issue by ghostdevv** [roxiness/routify#356]
Taken from [here](https://github.com/roxiness/routify/issues/191)

**Comment by jakobrosenberg** [comment]
V3 feature?

**Comment by ghostdevv** [comment]
@jakobrosenberg Yes, but also not just a ignore file maybe having `_` at the start of the file name or a routify option can also cause a ignore of the file

EDIT: Unsure about the `_` but it's something to think about
